src/pages/home/report/ReportActionItem.js

```diff
@@ -131,13 +131,6 @@ class ReportActionItem extends Component {
                                 onClose={this.hidePopover}
                                 anchorPosition={this.popoverAnchorPosition}
                                 animationIn="fadeIn"
-                                measureContent={() => (
-                                    <ReportActionContextMenu
-                                        isVisible
-                                        reportID={-1}
-                                        reportActionID={-1}
-                                    />
-                                )}
                             >
                                 <ReportActionContextMenu
                                     isVisible={this.state.isPopoverVisible}
```

Comment:
Ahhh I believe that in order to get rid of the `measureContent` prop you need to pass `isVisible` hard-coded as true on this line or the popover measurement won't work.

Reply:
isVisible isn't actually being used to make this render though, it's just for focusing the input. In that case it should still measure correctly right?

Reply:
> `isVisible` isn't actually being used to make this render though

Hmmm I'm a bit confused by this comment:

```jsx
const ReportActionContextMenu = (props) => {
    const wrapperStyle = getReportActionContextMenuStyles(props.isMini);
    return props.isVisible && (
        <View style={wrapperStyle}>
            {CONTEXT_ACTIONS.map(contextAction => (
                <ReportActionContextMenuItem
                    icon={contextAction.icon}
                    text={contextAction.text}
                    isMini={props.isMini}
                    key={contextAction.text}
                />
            ))}
        </View>
    );
};
```

As you can see, if `isVisible` is false, the `ReportActionContextMenu` won't render anything. Therefore, due to RN optimizations, it won't be included in the component tree at all, so we can't invisibly render it to measure it. Have you tested if this breaks the popover `ReportActionContextMenu`?

Reply:
Possibly dumb question.. but what does this change have to do with emojis?

Reply:
OH @roryabraham I thought this comment was on the `EmojiPickerMenu` component which _doesn't_ do this. I can change it.

Reply:
@marcaaron it isn't directly related, but since I was using PopoverWithMeasuredContent already and it was a fast refactor I removed the necessity for passing a measureContent prop

Reply:
> Have you tested if this breaks the popover ReportActionContextMenu

AFAICT it looks fine..